Package.swift

```diff
@@ -29,3 +30,22 @@ let package = Package(
             dependencies: ["SymbolKit"]),
     ]
 )
+
+// If the `SWIFTCI_USE_LOCAL_DEPS` environment variable is set,
+// we're building in the Swift.org CI system alongside other projects in the Swift toolchain.
+// In this case we only want to build the `dump-unified-graph` tool outside of Swift CI.
+if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
+    // Building standalone, so fetch all dependencies remotely.
+    package.dependencies += [
+        .package(url: "https://github.com/apple/swift-argument-parser", .upToNextMinor(from: "1.0.1")),
+    ]
+
+    package.targets += [
+        .executableTarget(
```

Comment:
I _think_ `executableTarget` wasn't introduced until around Swift 5.4 so I this is technically incompatible with the 5.2 package description here.

Maybe we should just only add the new target to the `5.5` package description for simplicity? Since the main goal of the the `5.2` one is backwards compatibility with packages that depend on SymbolKit.